### DISCUSSION TITLE
[New Component] [rich-text] 建议出个富文本组件

### DISCUSSION THREAD

**Discussion by LiYuhangggg** [element-plus/element-plus#17792] [Feature Request]
Feature type: **`New Component`**

## Details

- New component name: `el-rich-text`

### Features / 特性
1. 可以编辑文字、更改文字样式、发送表情、插入链接、上传图片、编辑图片、上传视频、插入html标签等功能。

### 已有的例子
https://www.wangeditor.com/  // 其他组件库的链接

### Additional comments

富文本也有其他的库，但我还是喜欢饿了么的东西

**Comment by elarbor** [comment]
还是[elementUI-vue3](https://element-plus.org/)用的惯，别的组件库我都用不惯😂
